tools/go-cli/go-whisk-cli/commands/package.go

```diff
@@ -185,16 +184,6 @@ var packageCreateCmd = &cobra.Command{
         }
         client.Namespace = qName.namespace
 
-        if (flags.common.shared == "yes") {
```

Comment:
I had understood the plan as making yes/no optional. I believe with this change, once you make the package public (or other types), you cannot make it private. Is that a correct statement?

Note that the python CLI allows yes/no optionally but requires --shared no to make the package private.
To wit, we need a test that makes a public package private via CLI.